index.bs

```diff
@@ -446,6 +446,359 @@ A file containing a 'pict' track compliant with this profile is expected to list
+            <td>0, 1</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to indicate directional relationships between images or metadata</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>auxl</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is auxiliary to another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>thmb</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is a thumbnail of another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>dimg</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is derived from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>prem</td>
+            <td>-</td>
+            <td>MIAF</td>
+            <td>&nbsp;</td>
+            <td>Used when when an alpha image contains premultiplied color values from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>cdsc</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used to link metadata with an image</td>
+        </tr>
+        <tr>
+            <td>idat</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to store derived image definitions</td>
+        </tr>
+        <tr>
+            <td>grpl</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to indicate that multiple images are semantically grouped</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>altr</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>&nbsp;</td>
+            <td>Used when images in a group are alternative to each other</td>
+        </tr>        
+        <tr>
+            <td>pasp</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>ipco</td>
+            <td>Used to signal pixel aspect ratio. If present, shall indicate a pixel aspect ratio of 1:1</td>
+        </tr>
+        <tr>
+            <td>colr</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>ipco</td>
+            <td>Used to signal information such as color primaries.</td>
+        </tr>
+        <tr>
+            <td>auxC</td>
+            <td>&nbsp;</td>
+            <td>0</td>
+            <td>HEIF</td>
+            <td>ipco</td>
+            <td>Used to signal the type of an auxiliary image (e.g. alpha, depth).</td>
+        </tr>
+        <tr>
+            <td>clap</td>
+            <td>&nbsp;</td>
+            <td>0</td>
```

Comment:
ISOBMFF lists `clap` as being a simple box. This should probably be `-`.

Reply:
Fixed